src/components/WorkspaceMembersPlaceholder.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+import Text from './Text';
+import OptionsListSkeletonView from './OptionsListSkeletonView';
+
+const propTypes = {
+    dataLoaded: PropTypes.bool,
+    dataEmptyText: PropTypes.string,
+};
+
+const defaultProps = {
+    dataLoaded: false,
+    dataEmptyText: undefined,
+};
+
+function WorkspaceMembersPlaceholder(props) {
```

Comment:
NAB: prefer destructuring https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#destructuring

```suggestion
function WorkspaceMembersPlaceholder({dataLoaded, dataEmptyText}) {
```